### PR TITLE
sunset 4.13 and adjust the cron time for ROSA with OCP e2e testing

### PR DESCRIPTION
**Pull request by yasun1**
Follow up on [OCM-11453](https://issues.redhat.com//browse/OCM-11453) to sunset 4.13 jobs and adjust the cron times for >4.13 jobs. 

I do not adjust 4.17 stable jobs for there is no 4.18 stable jobs. let us do it after the job is copied.

### CODE REVIEW

**Comment by yasun1** [comment]
@liangxia Could you help approve the change? Thanks.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml [hunk 1]
```diff
@@ -32392,18 +32392,7 @@ periodics:
     ci-operator.openshift.io/variant: amd64-stable
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-aws-rosa-hcp-byo-kms-oidc-auditlog-stage-critical-f14
-  reporter_config:
-    slack:
-      channel: '#managed-hypershift-ci-watcher'
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:32398]: Slack notification got removed here. Is it expected ?

Comment (yasun1) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:32398]: No, but it is inconvenient to handle this manually.  Just restore them back.

**Comment by yasun1** [comment]
@liangxia Please help review again.

**Pull request merged**